common/src/main/java/tachyon/security/UserInformation.java

```diff
@@ -114,10 +146,89 @@ static String getOsPrincipalClassName() {
 
     @Override
     public AppConfigurationEntry[] getAppConfigurationEntry(String appName) {
-      // TODO: switch branch based on appName. (Simple, Kerberos, etc)
-      return SIMPLE;
+      if (appName.equalsIgnoreCase(AuthenticationFactory.AuthTypes.SIMPLE.getAuthName())) {
+        return SIMPLE;
+      } else if (appName.equalsIgnoreCase(AuthenticationFactory.AuthTypes.KERBEROS.getAuthName())) {
+        // TODO: return KERBEROS;
+        throw new UnsupportedOperationException("Kerberos is not supported currently.");
+      }
+      return null;
+    }
+  }
+
+  /**
+   * Set the static configuration and initialize based on it
+   * @param conf the configuration
+   */
+  public static void setTachyonConf(TachyonConf conf) {
```

Comment:
Do you have this public static method only for unit-test purpose? If yes, then this is not necessary we have alternative solutions to work around; if no, I am afraind this is probably not how we are suppoed to use a static member as it can be set by an arbitrary external class.